pbdf/Issues/email/description.xml

```diff
@@ -25,24 +25,24 @@
+		<en>Add your email address to your IRMA app. You can add multiple email addresses.</en>
+		<nl>Laad je e-mailadres in je IRMA-app. Je kunt meerdere e-mailadressen toevoegen.</nl>
     	</FAQIntro>
 	<FAQPurpose>
-		<en></en>
-		<nl>Met jouw e-mailadres in de IRMA-app kun je laten zien dat dit e-mailadres echt van jou is.\n\nJe kunt bijvoorbeeld je e-mailadres gebruiken om op een website in te loggen. Of je kunt je e-mailadres aan een organisatie doorgeven.</nl>
+		<en>With your email address in your IRMA app, you can show that this address is really yours. For instance, you can use your email address to fill in a line in a form, or to login at a website.</en>
+		<nl>Met jouw e-mailadres in de IRMA-app kun je laten zien dat dit adres echt bij jou hoort. Je kunt bijvoorbeeld je e-mailadres gebruiken om een regel in een formulier in te vullen of om op een website in te loggen.</nl>
 	</FAQPurpose>
 	<FAQContent>
-		<en></en>
-		<nl>Je haalt de volgende gegevens op:\n• e-mailadres\n• e-mail domeinnaam</nl>
+		<en>You will add the following data.\n• emailaddress\n• email domain name</en>
+		<nl>Je laadt de volgende gegevens.\n• e-mailadres\n• e-mail domeinnaam</nl>
 	</FAQContent>
 	<FAQHowto>
-		<en></en>
-		<nl>Doorloop de volgende stappen:\n1. Vul je e-mailadres in.\n 2. Open je e-mailprogramma en ga naar 'inkomende berichten'.\n 3. Open de link in het e-mailbericht van Stichting Privacy by Design.\n 4. Je wordt teruggestuurd naar de IRMA-app.\n\nLet op\nAls je de e-mail op een computer opent, krijg je een QR-code te zien.\nDoorloop dan de volgende stappen:\n1. Open de link in het e-mailbericht van Stichting Privacy by Design.\n 2. Start de QR-scanner in de IRMA-app.\n 3. Scan de QR-code die op het scherm van je computer staat.</nl>
+		<en>Complete the following steps.\n1. Enter your e-mail address.\n2. Open your email application and go to your Inbox (if there is no new mail soon, please check your Spam folder).\n3. Click the link in the email message, sent by the Privacy by Design Foundation. If you see a QR code on your computer screen, scan it with the QR scanner in your IRMA app. The Privacy by Design Foundation copies your email data into your IRMA app.\n\nYou now receive a card with your email address and its domain name in your IRMA app. Repeat this process for other email addresses that you wish to add.</en>
+		<nl>Doorloop de volgende stappen.\n1. Vul je e-mailadres in.\n2. Open je e-maillezer en ga naar 'inkomende berichten' (als er geen mail verschijnt, controleer zonodig je Spam folder).\n3. Open de link in het e-mailbericht van de stichting Privacy by Design. Als je een QR-code te zien krijgt, scan deze met je IRMA-app. De stichting Privacy by Design voegt je e-mail gegevens toe in je IRMA-app.\n\nJe ontvangt nu een kaartje met je e-mailadres en de domeinnaam van dat adres in je IRMA app. Herhaal dit proces voor ieder e-mail adres dat je toe wil voegen.</nl>
```

Comment:
It's a bit subtle and contentious, but I suggest it either is `Stichting Privacy by Design` or `Privacy by Design`.

Currently, we don't have a name in the email `From` address. We should change that, and this text should reflect that `From` name. I'd suggest `Stichting Privacy by Design`.

Reply:
@bajacobs I agree with @confiks . Shall I change it accordingly?

Reply:
I feel rather strongly about keeping "de" before stichting. It makes the sense run more smoothly in Dutch.

On a slightly different note, I have tried to be consistent in writhing "stichting Privacy by Design" and "Privacy by Design foundation" where both stiichting and foundation start with a small letter (s and f). But I see that the english text here has Foundation with F. @CreativeCode can you double-check (grep) on this throughout all texts?

Reply:
At least for English, be aware that nobody else does this, and styleguides deem it incorrect. Do you really propose for example also changing this header? It looks like a typo to me.

<img width="1027" alt="Screen Shot 2020-03-25 at 12 14 21 AM" src="https://user-images.githubusercontent.com/7931323/77485538-afc92480-6e2d-11ea-9570-9d1099702862.png">

Reply:
What is "this" in your first line? Is there a misunderstanding? I don't want to change the header since it is precisely as I propose: "foundation" with "f"

Reply:
Currently we use "Privacy by Design Foundation" (capitalized) everywhere. The screenshot above is a representation of the change you are proposing.

Reply:
So I'm primarily talking about titles and email `From` addresses specifically, not prose.

Reply:
Indeed, I propose to change it everywhere in the scheme explanations.

Reply:
It's unclear if you want the website header and From address changed as well. I'll leave this to @CreativeCode.